Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -149,6 +149,10 @@
                              cosmicDCTracksSeq
                              )
 
+# LowPtGsfElectrons
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronSequence_cff import *
+highlevelreco *= lowPtGsfElectronSequence
```

Comment:
please keep this explicitly in highlevelreco

Reply:
I interpret this to mean "leave untouched" ...?

Reply:
No, I imagine that the suggestion was to move lowPtGsfElectronSequence explicitly inside highlevelreco, a few lines above in the same config
Making it separately makes sense if you want to add conditionally (e.g. for a given Era), not if you want to add it in any case.

Reply:
Done, [here](https://github.com/cms-sw/cmssw/pull/25455/commits/4f17a824e13c9afe1f56e1c6c31b812542116816)

Reply:
I squashed the above commit into [here](https://github.com/cms-sw/cmssw/pull/25455/commits/7ef049af1381373f1a65ff0a65356ffb32620b4f#diff-bc1d3eddbea8b75d590d62e72fee751e).